admin/sql/create_tables.sql

```diff
@@ -18,4 +18,22 @@ CREATE TABLE listen (
   raw_data        JSONB
 );
 
+CREATE TABLE token (
+     id               SERIAL,
+     user_id          INTEGER, -- FK to "user".id
+     token            TEXT NOT NULL,
+     api_key          VARCHAR NOT NULL,
+     ts               TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+ALTER TABLE token ADD CONSTRAINT token_api_key_uniq UNIQUE (api_key);
```

Comment:
I already have a index on `(token, api_key)` which would be used for lookups. But each token has to be UNIQUE, that is why i added a UNIQUE constraint on this column. Probably this is a better way to do this,
```
CREATE TABLE token (                                                                                                                                                    
     ... 
     token            TEXT NOT NULL UNIQUE,
     ...
);
```